src/components/profile/static/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+// Components
+import ProfileDetails from './ProfileDetails';
+
+// MUI
+import Paper from '@material-ui/core/Paper';
+import withStyles from '@material-ui/core/styles/withStyles';
+import style from '../../../style/style';
```

Comment:
style/style? can you make the last one an index.js so you just import it with one /style?

Reply:
Fixed